MANUAL.txt

```diff
@@ -3377,6 +3377,14 @@ In this example, the document will contain a citation for `item3`
 only, but the bibliography will contain entries for `item1`, `item2`, and
 `item3`.
 
+It is possible to create a bibliography with all the citations, 
+whether or not they appear in the document, by using a wildcard:
+
+    ---
+    nocite: |
```

Comment:
I had a similar thought and tested with the single quotes, which worked. I've been using this in a bash script for a while now, so I was pretty sure it was ok. I was going on your suggestion, actually, @jgm : https://groups.google.com/d/msg/pandoc-discuss/mp0PoYmWtGk/m24UwCeyBgAJ

That said, it seems like the unquoted version works as well...though I'm not sure there aren't circumstances where the quoting is needed.